toolchain/lower/file_context.h

```diff
@@ -73,6 +76,8 @@ class FileContext {
   // the caller.
   auto BuildType(SemIR::InstId inst_id) -> llvm::Type*;
 
+  auto BuildGlobalVariableDecl(SemIR::InstId inst_id) -> llvm::GlobalVariable*;
```

Comment:
We're generally trying to have brief comments on functions, how about something similar to the above:

```suggestion
  // Builds the global for the given instruction, which should then be cached by the caller.
  auto BuildGlobalVariableDecl(SemIR::InstId inst_id) -> llvm::GlobalVariable*;
```

Reply:
Done.